### ISSUE TITLE
Change the current file

### ISSUE THREAD

**Issue by inacionery** [srstevenson/nb-clean#46]
Hey

Is it possible to change the current file by running `nb-clean -i Sample.ipynb`? Instead of creating a new one and renaming it to the old one.

Thank you!

By the way, congratulations for this cool tool!

**Comment by srstevenson** [comment]
It's a bit cumbersome, but you can do this using [`sponge`](https://manpages.debian.org/buster/moreutils/sponge.1.en.html#NAME) (from [`moreutils`](https://joeyh.name/code/moreutils/), which is [widely packaged](https://repology.org/project/moreutils/versions)):
```sh
nb-clean clean -i Sample.ipynb | sponge Sample.ipynb
```

**Comment by srstevenson** [comment]
@inacionery This spurred some redesign and refactoring, and I've just released [2.0.0](https://github.com/srstevenson/nb-clean/releases/tag/2.0.0) which operates in place by default, meaning you can now use:
```bash
nb-clean clean Sample.ipynb
```

**Comment by inacionery** [comment]
@srstevenson thank you!
